test/tests/functional/pbs_qmgr.py

```diff
@@ -98,11 +100,19 @@ def test_listcmd_whitespaces(self):
             long_string += blah
             # Create a new vnode
             attrs = {ATTR_rescavail: 2}
-            vndef = self.mom.create_vnode_def("vn", attrs)
+            vndef = self.mom.create_vnode_def(node_prefix, attrs)
```

Comment:
If the purpose here is just to add one vnode then please use self.server.create_vnodes(nodename, attrs, 1, self.mom). This will make sure that vnode is created with attributes you specified.
If you do this then you don't have to check for mom to be up